_posts/2018-05-24-open-source-hackillinois-2018.markdown

```diff
@@ -12,25 +12,25 @@ Open Source carried a huge presence at HackIllinois 2018. This year, we had over
 
 To initially introduce students to the mentors, we held panels regarding Web/Data and Systems/Miscellaneous so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
 
 ![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/open-source-hackillinois-2018/panel_speakers.jpg "Web/Data panel")
 
-Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand, and were prepared to guide students through the initially intimidating process of understanding a codebase and improving it. 
+​Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand and were prepared to guide students through the initially intimidating process of understanding and improving a codebase.
 
 ![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/open-source-hackillinois-2018/powershell.jpg "Apache Mesos")
 
 Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize]({{ site.baseurl }}{% post_url 2017-09-29-open-source-at-hackillinois-rust-cookbook %}) for HackIllinois 2017. This year, he came back with other students from Indiana University–Purdue University Indianapolis (IUPUI) and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long-term relationships between students and the Open Source community, and we're super glad that groups from HackIllinois are still working together to contribute to and build on projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [RustBridge](https://github.com/rustbridge).
 
 ![Hackers from IUPUI work with mentors from Rust in ECEB](/assets/article_images/open-source-hackillinois-2018/rust.jpg "Rust")
 
-We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing and had mentor Carol Willing (former Director of the Python Software Foundation) returning from HackIllinois 2017 to work on Jupyter.
+We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing. Of course, we also had mentor Carol Willing (former Director of the Python Software Foundation) returning from HackIIllinois 2017 to work on [Jupyter](https://github.com/jupyter/jupyter).
```

Comment:
This whole paragraph sounds very passive. This might read a little better:
"We also had some awesome mentors representing Open Source Data Science projects. Jeff Bezanson, creator of Julia, worked with hackers on solving issues for the Julia language. Carol Willing (former Director of the Python Software Foundation) joined us for another year and worked with hackers on the Jupyter Project"

Another possible rewrite:
"Jeff Bezanson, creator of Julia, worked with hackers on solving issues for the Julia language, and Carol Willing (former Director of the Python Software Foundation) joined us for another year and worked with hackers on the Jupyter Project"